examples/src/main/scala/org/apache/spark/examples/ml/LogisticRegressionSummaryExample.scala

```diff
@@ -30,11 +30,11 @@ object LogisticRegressionSummaryExample {
   def main(args: Array[String]): Unit = {
     val conf = new SparkConf().setAppName("LogisticRegressionSummaryExample")
     val sc = new SparkContext(conf)
-    val sqlCtx = new SQLContext(sc)
-    import sqlCtx.implicits._
+    val sqlContext = new SQLContext(sc)
+    import sqlContext.implicits._
```

Comment:
@srowen I searched all examples in ml and corrected them if they are used in examples in the document. There is one logisticRegression training summary example, which is not part of the document. So I think it is not necessary to change it. Thanks!